packages/material-ui/src/ButtonBase/focusVisible.js

```diff
@@ -47,11 +47,11 @@ const handleKeyUpEvent = event => {
   if (isFocusKey(event)) {
     internal.focusKeyPressed = true;
 
-    // Let's consider that the user is using a keyboard during a window frame of 1s.
+    // Let's consider that the user is using a keyboard during a window frame of 500ms.
     clearTimeout(internal.keyUpEventTimeout);
     internal.keyUpEventTimeout = setTimeout(() => {
       internal.focusKeyPressed = false;
-    }, 1e3);
+    }, 5e2);
```

Comment:
Do we really need scientific notation?

Reply:
500?

Reply:
Personally i would use the `ms` notation as i feel that is more familiar for people reading the code, but wanted to keep notation consistency, hence the `5e2`.